erpnext/templates/job-configure-bench.yaml

```diff
@@ -46,6 +46,9 @@ spec:
             [[ -f common_site_config.json ]] || echo "{}" > common_site_config.json;
             bench set-config -gp db_port $DB_PORT;
             bench set-config -g db_host $DB_HOST;
+            {{- if .Values.dbRds }}
+            bench set-config -g rds_db 1;
```

Comment:
-gp we'll be needed instead of just -g.

-p makes sure 1 remains integer. or it'll become "1" string.